frontend/src/features/admin/agencies/ManageAgencies.test.tsx

```diff
@@ -66,30 +67,28 @@ const getStore = () =>
   });
 
 describe('Manage Agencies Component', () => {
+  beforeAll(() => {
+    const { getComputedStyle } = window;
```

Comment:
this just fixes test warnings, no functional changes. see https://github.com/nickcolley/jest-axe/issues/147